src/winetricks

```diff
@@ -12292,6 +12292,97 @@ load_pngfilt()
 
 #----------------------------------------------------------------
 
+w_metadata powershell_core apps \
+    title="PowerShell Core" \
+    publisher="Microsoft" \
+    year="2024" \
+    media="download" \
+    file1="PowerShell-7.2.19-win-x86.msi" \
+    file2="PowerShell-7.2.19-win-x64.msi"
+    
+
+
+# Function to download and install PowerShell Core
+load_powershell_core()
+{
+    # Uncomment below and remove win32 download elif and file when PowerShell Core v7.2 LTS goes EOL
+    #w_package_unsupported_win32
+
+    # Derive version number from file1
+    _load_powershell_core_regex='[0-9]+\.[0-9]+\.[0-9]+'
+
+    if echo "${file1}" | grep -E -o "${_load_powershell_core_regex}" >/dev/null; then
+        _load_powershell_core_version=$(echo "${file1}" | grep -E -o "${_load_powershell_core_regex}")
+    fi
+
+    # Download PowerShell Core 7.2.x MSI (Latest LTS Version to support win32)
+    if [ "${W_ARCH}" = "win64" ]; then
+        # https://github.com/PowerShell/PowerShell/releases/v7.2.19
+        w_download "https://github.com/PowerShell/PowerShell/releases/download/v${_load_powershell_core_version}/${file2}" 965e10fc77dbce6ef622f1c0fcddc03bebd2b3629f1a3253cc17a8ffc71a6522
```

Comment:
Please don't use variables with w_download URLs, it breaks src/linkcheck.sh.

BTW, if you run ./tests/shell-checks before submitting, you'll get an error letting you know that.

Reply:
Good to know, I have fixed this as of https://github.com/Winetricks/winetricks/pull/2211/commits/b51fbac1d52db177162b5a87425d1c43da1dff7a

./tests/shell-checks appears to be happy after running a test.